src/dialog.scss

```diff
@@ -31,6 +31,8 @@ $button: #{$fd-namespace}-button;
   $fd-dialog-content-box-shadow: var(--sapContent_Shadow3);
   $fd-dialog-content-border-radius: var(--sapElement_BorderCornerRadius);
   $fd-dialog-content-padding-x: (def: 1rem, s: 1rem, m: 2rem, l: 2rem, xl: 3rem);
+  $fd-dialog-content-min-width-x: (def: 0, s: 0, m: 600px, l: 1024px, xl: 1440px);
+  $fd-dialog-content-max-width-x: (def: 599px, s: 599px, m: 1023px, l: 1439px, xl: 0);
```

Comment:
Where do this vaules come from?
I talked to the designer once, and they said there are no predefined sizes for the dialog. I just went through the specs and I also couldn't find any information.

Reply:
Oh I saw it in the fd-styles documentation for dialog and didn't think to cross check it with the specs: https://sap.github.io/fundamental-styles/?path=/docs/components-dialog--default-dialog

If there's no defined sizes, why have size modifiers at all? The spec for the PDF viewer indicates a size `l` dialog as well. I can just set a width for the PDF viewer and revert these changes then.

Reply:
@salarenko Any further input on this topic? The rest of the PR is ready to go. I can either remove these widths or remove the part of the docs that specifies these widths.
I checked the UX spec and I'm still not clear on what the sizes are supposed to refer to.

Reply:
@meganaconley 
- I believe the section you are referring to in the Dialog documentation is describing what classes be used depending on the Dialog size, to add proper margins to the Dialog window. There is nothing said about "size modifiers".

- In Fiori Desgin Guideliness: "The PDF viewer control is fully responsive on large-screen devices (size L).". Again there is nothing said about the Dialog size, but rather about the size of the device screen.

I totally agree that Fiori Fundamentals should have something like predefined Dialog sizes, but there is nothing about it in the design, and the designers are not very helpful with this topic... Looking at the Fiori Guideliness I'd rather say that the Dialog should be opened in `width: 100; height: 100%`.
And you see, all of this are only assumptions and because of that I'm not sure if we should even have this component, because it has not been described in "Fiori VIsual Design".